smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -78,6 +78,39 @@ def test_add_remove_member(self):
             self.session.delete(self._base_path + [interface, 'member'])
 
         self.session.commit()
+    
+    def test_vlan_filter(self):
+        """ Add member interface to bridge and set VLAN filter """
+        for interface in self._interfaces:
+            base = self._base_path + [interface]
+            self.session.set(base + ['address', '192.0.2.1/24'])
+            self.session.set(base + ['vif', '2','address','192.0.3.1/24'])
+
+            vlan_id = 101
+            # assign members to bridge interface
+            for member in self._members:
+                base_member = base + ['member', 'interface', member]
+                self.session.set(base_member + ['allowed-vlan', '2'])
+                self.session.set(base_member + ['allowed-vlan', '4-9'])
+                self.session.set(base_member + ['native-vlan', str(vlan_id)])
+                vlan_id += 1
+
+        # commit config
+        self.session.commit()
+
+        # check member interfaces are added on the bridge
+        bridge_members = []
+        for tmp in glob(f'/sys/class/net/{interface}/lower_*'):
+            bridge_members.append(os.path.basename(tmp).replace('lower_', ''))
+
+        for member in self._members:
```

Comment:
Please add validation logic that the vlan interfaces have been created and also the allowed-vlan and native-vlan is properly setup. Right now this only tests the no error is generated by CLI but not if the functionality actually works.